tests/observability/network/conftest.py

```diff
@@ -177,7 +180,7 @@ def invalid_cnao_operator(prometheus, admin_client, hco_namespace, csv_scope_cla
         patches={csv_scope_class: bad_cnao_operator},
         list_resource_reconcile=[NetworkAddonsConfig],
     ):
-        yield
+        yield csv_scope_class
```

Comment:
please update the fixture name - it patches the csv (and now also yields it) to something like 
`csv_with_invalid_cnao_operator`